### PR TITLE
Feature/custom cell nodes

### PR DESCRIPTION
**Pull request by vadymmarkov**
@zenangst @RamonGilabert @richardoti 
Sorry for a big PR, but it's kind of breaking change. 
1) According to https://github.com/hyperoslo/Wall/issues/60 we decided to go without a huge Config, but with the possibility to register your custom cell or cell subview (header, footer, action bar).
2) Simplify things, remove some models (group, location). The idea is to have some basic default functionality that could be easily extended.
3) Refactor code.

The example looks almost the same as before:
ios simulator screen shot 03 aug 2015 09 09 41

### CODE REVIEW

File: Source/Nodes/Defaults/ActionBarNode.swift [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      attributes: [
+        NSFontAttributeName: UIFont.boldSystemFontOfSize(14),
+        NSForegroundColorAttributeName: UIColor.lightGrayColor()
+      ])
+
+    let node = ControlNode(title: title)
+    node.userInteractionEnabled = true
+
+    return node
+  }()
```
Comment (RamonGilabert) [inline Source/Nodes/Defaults/ActionBarNode.swift]: The spacing here it's wrong I think, and in all the properties below

File: Source/Nodes/Defaults/CellNode.swift [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+    node.attributedString = NSAttributedString(
+      string: self.post.text,
+      attributes: [
+        NSFontAttributeName: UIFont.systemFontOfSize(14),
+        NSForegroundColorAttributeName: UIColor.grayColor()
+      ])
+    node.userInteractionEnabled = true
+
+    return node
+  }()
```
Comment (RamonGilabert) [inline Source/Nodes/Defaults/CellNode.swift]: Also here

**Comment by RamonGilabert** [comment]
Nice work man!

**Comment by vadymmarkov** [comment]
@RamonGilabert Fixed spacings.

**Comment by vadymmarkov** [comment]
@RamonGilabert Thanks man!

**Pull request merged**